geode-core/src/main/java/org/apache/geode/cache/client/internal/pooling/ConnectionManagerImpl.java

```diff
@@ -70,6 +70,10 @@
 public class ConnectionManagerImpl implements ConnectionManager {
   private static final Logger logger = LogService.getLogger();
   private static final int NOT_WAITING = -1;
+  public static final String BORROW_CONN_ERROR_MSG = "Could not create a new connection to server ";
+  public static final String UNEXPECTED_SOCKET_CLOSED_MSG =
+      "Pool unexpected closed socket on server";
+  public static final String SOCKET_TIME_OUT_MSG = "socket timed out on client";
```

Comment:
Its good to use this variable while we create the exception message; that way any changes to the message is reflected in all the places.
In OpExecutorImpl:
else if (e instanceof SocketTimeoutException) {
      invalidateServer = timeoutFatal;
      title = "socket timed out on client"; 
      cause = null;
    } e